app/src/main/java/org/oppia/app/onboarding/OnboadingSlideViewModel.kt

```diff
@@ -22,25 +25,29 @@ class OnboardingSlideViewModel(val context: Context, viewPagerSlide: ViewPagerSl
   private fun initializingSlide(viewPagerSlide: ViewPagerSlide) {
     when (viewPagerSlide) {
       ViewPagerSlide.SLIDE_0 -> {
-        slideImage.set(R.drawable.ic_onboarding_0)
+        if(orientation == Configuration.ORIENTATION_LANDSCAPE){slideImage.set(R.drawable.ic_landscape_onboarding_2)}
```

Comment:
The final/image numbering is incorrect. The first slide should have 0 index, second slide should have 1 index and so on, similar to previous images. Here it should be `ic_onboarding_landscape_0`

Reply:
Done